Lotto/Lotto/StringCalculator/StringConverter.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  StringConverter.swift
+//  Lotto
+//
+//  Created by brian은석 on 2022/05/08.
+//
+
+import Foundation
+
+struct StringConverter {
+    func split(_ input: String?) -> [String] {
+        guard let input = input else { return [""] }
+        guard !input.isEmpty else { return [""] }
+        
+        return input.components(separatedBy: [",",":"])
+    }
+    
+    func emptyValueChangeZero(inputArray: [String]) -> [String] {
+        var changeArray: [String] = []
+        for value in inputArray {
+            changeArray.append(emptyValueChangeZero(input:value))
+        }
+        return changeArray
+    }
+    
+    private func emptyValueChangeZero(input: String) -> String {
+        if input.isEmpty {
+            return "0"
+        }
+        return input
+    }
+    
+    func stringToInt(inputArray: [String]) throws -> [Int] {
```

Comment:
입력값과 반환값이 배열이네요, 메서드 명에서 이걸 나타나게 할수는 없을까요?
```suggestion
    func stringsToInts(input: [String]) throws -> [Int] {
```
이건 어떨까요?

Reply:
다음 스텝에 반영해서 커밋하겠습니다!